### DISCUSSION TITLE
Color subview backgrounds

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5634] [Feature Requests]
The ability to change the background color of subviews would be really nice.

**Requested By:** NMC Herbaria

**Comment by tlammer** [comment]
Specify 6 does this through the xml form code by adding bgcolor=x,x,x to the initialized phrase.
So this:
                    <cell type="subview" viewname="Determination" id="6" name="determinations" colspan="14" rows="5" initialize="bgcolor=150,20,50"/>

produces a burgundy determination subform 
![image](https://user-images.githubusercontent.com/12532430/188721178-92905ba6-431c-41d6-9176-3ea8aeb58973.png)
